gulpfile.js

```diff
@@ -94,45 +91,28 @@ const allJs = gulp.parallel(
 
 function copy() {
   return gulp.src([
-    // Copy all files
-    'src/**',
-    // Include the .well-known folder
-    'src/.**/*',
-    // Exclude the following files
-    // (other tasks will handle the copying of these files)
-    '!src/*.html',
-    '!src/{css,css/**}',
-    '!src/{js,js/**}'
+    'src/{.well-known,imgs,test-svgs}/**',
+    // Exclude the test-svgs files except for `car-lite.svg`
+    // which is used in the demo
+    '!src/test-svgs/!(car-lite.svg)',
```

Comment:
I had to use this weird-looking expression to get to keep only car-lite.svg

Reply:
We can just remove the other test SVGs to avoid this

Reply:
Your call, I find some of the other test files good to have, but now they won't be deployed anyway. Maybe just keep them for historical reasons?

Reply:
Another solution would be to output the demo SVG to the build folder; that would also get rid of this workaround.

BTW the demo SVG could also probably be in the SW; it's not that big and would probably simplify some of the JS code.

Reply:
Happy to land this as-is. Can always split this into `demo.svg` and `test-svgs` later (where the latter isn't part of the build).